cli/cli.ts

```diff
@@ -2091,7 +2091,7 @@ function buildTargetCoreAsync(options: BuildTargetOptions = {}) {
                         pxt.appTarget.simulator.dynamicBoardDefinition)
                         isPrj = true
                 })
-                .then(() => options.quick ? null : testForBuildTargetAsync(isPrj || (!options.skipCore && isCore), builtInfo[dirname] && builtInfo[dirname].sha))
+                .then(() => options.quick || config.partial ? null : testForBuildTargetAsync(isPrj || (!options.skipCore && isCore), builtInfo[dirname] && builtInfo[dirname].sha))
```

Comment:
but this seems to prevent build of any partial project; it's during the build that we generate enums.d.ts

Reply:
looking

Reply:
always testForBuild now